lagom-cloud-object-storage-example/README.md

```diff
@@ -0,0 +1,100 @@
+
+1.  [Download and set up the Lagom service](#download-and-set-up-the-lagom-service)
+2.  [Start the Lagom sample application](#start-the-lagom-sample-application)
+3.  [Generate some events on the Lagom service](#generate-some-events-on-the-lagom-service)
+4.  [Check uploaded files on Cloud Object Storage GUI](#check-cloud-object-storage)
+
+
+## Download and set up the Lagom service
+
+Follow these steps to get a local copy of this project and configure it with the Cloud Object Storage credentials and settings.
+
+1.  Open a command line shell and clone this repository:
+    ```
+    git clone https://github.com/lagom/ibm-integration-examples.git
+    ```
+2.  Change into the root directory for this example:
+    ```
+    cd lagom-cloud-object-storage-example
+    ```
+3.  Supply the configuration:
+    1. Copy the `account-impl/src/main/resources/cloud-object-storage.conf.template` file to `account-impl/src/main/resources/cloud-object-storage.conf`.
+    2. Open `account-impl/src/main/resources/cloud-object-storage.conf` in a text editor and fill in the necessary information. Details are provided in the file itself.
+
+## Start the Lagom sample application
+
+In the command line shell where you downloaded the Lagom service, from the `lagom-cloud-object-storage-example` directory, start the Lagom development environment by running:
+
+```
+mvn lagom:runAll
+```
+
+You should see some console output, including these lines:
+
+```
+...
+[INFO] Service gateway is running at http://localhost:9000
+...
+[INFO] (Service started, press enter to stop and go back to the console...)
+```
+
+These messages indicate that the service has started correctly.
+
+
+## Generate some traffic on the Lagom service
+
+This application simulates a bank account application where deposits and withdraws can be executed on an account. On every 5 transactions an account extract is generated and uploaded to Cloud Object Storage. Extracts can be downloaded for local visualization. 
+
+Account transactions are propagated from the write-side (`AccountEntity`) to the read-side (`AccountExtractProcessor`) as events stored in a Cassandra database. 
+
+Extracts are kept in-memory until they reach 5 transactions when they are uploaded to Cloud Object Storage. For the sake of simplicity, the `AccountExtractRepository`, that holds extracts in-memory, is not thread-safe and therefore its code is only suitable for demonstrations.
+
+This application doesn't have a GUI. Only a REST API. Any REST client or http tool can be used to interact with it. We provide a simple `api.sh` that can be used as a shell client that uses [HTTPie](https://httpie.org/).  
+
+The rest of this guide will use `curl` syntax to document the calls. You can adapt it to your REST client of choice or use the provided `api.sh` script. You will find detailed information about the calls for `HTTPie` in the file itself.
+
+  1. Source `api.sh` in your console (eg: `. api.sh`) - optional 
```

Comment:
The steps are confusing because it looks like you are presenting two different ways to do things. If the reader sources api.sh, do they need to type in the Curl calls? What exactly does sourcing api.sh do? 

this is what I suggest:

## Generate transactions and view stored extract
The example account number is 123-456-890. The Lagom service provides APIs to check the balance and to deposit or withdraw money. Use the REST calls below to create transactions. Be sure not to withdraw more money than the account balance.   Then, retrieve the extract from the Cloud Object Storage bucket.

To check the balance and generate transactions, use calls to the following endpoints:

1. To check the account balance: 'http://localhost:9000/api/account/123-4567-890/balance' For example:
    ```
    curl http://localhost:9000/api/account/123-4567-890/balance
    ```
2. To deposit money: 'http://localhost:9000/api/account/123-4567-890/deposit --data '{ "amount": 100 }' For example:
   ```curl -H "Content-Type: application/json" -XPOST http://localhost:9000/api/account/123-4567-890/deposit --data '{ "amount": 100 }'
    ```
3. To withdraw money ...(similar to above)

4. After the fifth operation ... (the rest of what is now step 5)

Check your Cloud Object Storage bucket (do we need to tell them how to do this?)  You should ...
Retrieve the account extract by calling .....

Reply:
I will rework it. I will also remove any mention to the `api.sh`. Although it's a really cool tool, it seems make the instructions less straightforward. 

I will leave the script where it is though. Curious developers may open it and find enough information inside, but I won't promote it on the README.

Reply:
@rstento with respect to explaining how to verify that the files were uploaded, I think that this must be explained by Cloud Object Storage documentation itself. 

In the list of requirements, we specify that a Cloud Object Storage account is needed. We assume that the user have setup an account and read the documentation of Cloud Object Storage.